Handler/Project.hs

```diff
@@ -1,4 +1,4 @@
-{-# LANGUAGE TupleSections, OverloadedStrings #-}
+{-# LANGUAGE CPP, TupleSections, OverloadedStrings #-}
```

Comment:
I probably added that, then realized that I didn't need it, forgot to remove it. This can get axed